tests/llm/test_example_selector.py

```diff
@@ -0,0 +1,218 @@
+import asyncio
+from typing import Any, Dict, List
+
+import pytest
+import numpy as np
+from langchain_core.example_selectors.base import BaseExampleSelector
+from langchain_core.example_selectors import LengthBasedExampleSelector
+from langchain_core.prompts import FewShotPromptTemplate, PromptTemplate
+from langchain_core.messages import AIMessage, HumanMessage
+from pydantic import BaseModel, Field, RootModel
+
+from chatsky import MISC, RESPONSE
+from chatsky.llm.example_selector import Example, StaticExampleSelector, to_langchain_context
+from chatsky.responses.llm import LLMResponse
+
+np.random.seed(0)
+
+
+class ExamplePrompt(BaseModel, arbitrary_types_allowed=True):
+    examples: StaticExampleSelector | BaseExampleSelector
+
+    async def __call__(self):
+        examples: list[dict[str, str]] = await self.examples.select_examples({"input": ctx.last_request.text})
```

Comment:
Maybe remove this method or raise `NotImplementedError`?
It doesn't do anything.